components/cam/src/physics/cam/qneg4.F90

```diff
@@ -1,6 +1,10 @@
 
 subroutine qneg4 (subnam  ,lchnk   ,ncol    ,ztodt   ,        &
+#ifdef MAML
+                  ncrm, qbot    ,srfrpdel,shflx   ,lhflx   ,qflx, excess    )
+#else
```

Comment:
I'm a bit confused why you did this when you could have just added:
#ifdef MAML
ncrm,
#endif

Reply:
Also, using "ncrm" on this general routine seems odd. I would suggest using the "subcolumn" terminology. That way we could get rid of some of the ifdefs below and just have a loop over the number of subcolumns, which would be 0 in the default case. But then again, that could lead to merge conflicts, so I'll leave that choice up to you.